File: Códigos de aula/string_as_array.py
frase = "Olá tudo bem com você"

# percorrer a frase em cada caracter
for c in frase:
    print(c)

# remover os acentos da frase
frase = frase.replace("á", "a")
frase = frase.replace("ê", "e")

print(frase)

File: Códigos de aula/string_as_array.c
#include <stdio.h>
#include <string.h>

int main()
{
    char frase[100];
    strcpy(frase, "Olá, tudo bem com você");
    int tam = strlen(frase);

    for (int i = 0; i < tam; i++)
    {
        printf("%c ", frase[i]);
    }
    return 0;
}
